### ISSUE TITLE
HINet: Half Instance Normalization Network for Image Restoration

### ISSUE THREAD

**Issue by dltmddbs100** [dltmddbs100/Paper-log#2]
## Main Idea
**What is best Normalization in low-level vision task? Half instance!**
Paper: <https://arxiv.org/abs/2105.06086>
## Abstract
 Training Deep Neural Networks is complicated by the fact that the distribution of each layer’s inputs changes during training, as the parameters of the previous layers change. This slows down the training by requiring lower learning rates and careful parameter initialization, and makes it notoriously hard to train models with saturating nonlinearities. We refer to this phenomenon as internal covariate shift, and address the problem by normalizing layer inputs. Our method draws its strength from making normalization a part of the model architecture and performing the normalization for each training mini-batch. Batch Normalization allows us to use much higher learning rates and be less careful about initialization. It also acts as a regularizer, in some cases eliminating the need for Dropout. Applied to a state-of-the-art image classification model, Batch Normalization achieves the same accuracy with 14 times fewer training steps, and beats the original model by a significant margin. Using an ensemble of batchnormalized networks, we improve upon the best published result on ImageNet classification: reaching 4.9% top-5 validation error (and 4.8% test error), exceeding the accuracy of human raters.

## Contribution
+ U-Nets의 구조를 사용한 multi-stage 방식으로써 SAM과 CSFF를 차용하여 설계한 HINET 제시
+ Instance normalization을 half part에만 적용, 나머지 half에 대해서는 channel의 정보들을 보존하는 half instance normalization block을 구축
+ 각 sub-network의 input image와 output을 더한 것을 ground truth를 비교하는 방식으로 PSNR loss를 정의
+ Half Instance Normalization의 직접적으로 사용해 low-level image task인 denoising, deblurring 등의 task에서 sota를 달성
+ Low-level task에서 발견되었던 normalization에 관한 한계를 극복, 효율적인 연산량과 추론시간의 단축
+ Normalization을 통해 Internal Covariant Shift 를 해소, 그에 따라 발생하는 학습 속도 및 성능의 저하를 해결 
## Reviews
<https://deepkerry.tistory.com/30>
